### DISCUSSION TITLE
Contributors List

### DISCUSSION THREAD

**Discussion by dhruvabhat24** [Lets-code-with-us/DSA-Cracker#15] [Announcements]
After contributing your first solution create a new Issue with tag contributor
Provide the below contents

1. Github Profile image for Top contributors list 
2. provide your username
3. Provide github profile ur 
4. Link ur first accepted pull request number

**Comment by dhruvabhat24** [comment]
> After contributing your first solution create a new Issue with tag contributor Provide the below contents
> 
> 1. Github Profile image for Top contributors list
> 2. provide your username
> 3. Provide github profile ur
> 4. Link ur first accepted pull request number

@sujan5757 @Abiji-2020 Kindly provide these deatils.

**Reply by Abiji-2020** [reply]
done the details are provided
